### ISSUE TITLE
How to generate tfmasks?

### ISSUE THREAD

**Issue by arunpatro** [zalandoresearch/disentangling_conditional_gans#2]
Hi,

While generating the tfrecords for the dataset, the code here: 
https://github.com/zalandoresearch/disentangling_conditional_gans/blob/03d5101ba63ed223d310e42584545bb1d53077e0/dataset.py#L79

loads *.tfmasks. But the code in the repo doesn't create tfmasks. Can someone share the format for tfmasks? Is it a binary valued image for each image? This can be created using segmentation but how to create a tfrecord/tfmask format of it?

Thank you.

**Comment by gokyildirim** [comment]
I will create an updated branch soon with an example toy dataset, I hope the dataset creation will be clearer.

**Comment by gokyildirim** [comment]
@arunpatro could you check the new branch "toydata", I hope it is clearer now.

**Comment by arunpatro** [comment]
Yes, the code is much easier to understand. Thank you so much. However I
see that train.py is different than master branch.

**Comment by gokyildirim** [comment]
yes, I've changed a few things on how the average color is handled. But both branches should lead to compatible trained models.
